{"uid":"ba850852bfa5b8ef","name":"test_full_cluster_restart[SIGKILL-postgres]","fullName":"tests.integration.ha_tests.test_self_healing#test_full_cluster_restart","historyId":"2702afd99304f2d258e13b03718c72a0","time":{"start":1728611650584,"stop":1728611692601,"duration":42017},"description":"This tests checks that a cluster recovers from a full cluster restart.\n\n    The test can be called a full cluster crash when the signal sent to the OS process\n    is SIGKILL.\n    ","descriptionHtml":"<p>This tests checks that a cluster recovers from a full cluster restart.</p>\n<pre><code>The test can be called a full cluster crash when the signal sent to the OS process\nis SIGKILL.\n</code></pre>\n","status":"broken","statusMessage":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 97, in pytest_runtest_teardown\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 100, in pytest_runtest_teardown\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 853, in pytest_runtest_teardown\n    yield from self._runtest_for(item, \"teardown\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 885, in pytest_runtest_teardown\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 189, in pytest_runtest_teardown\n    item.session._setupstate.teardown_exact(nextitem)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 557, in teardown_exact\n    raise exceptions[0]\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 546, in teardown_exact\n    fin()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1032, in finish\n    raise exceptions[0]\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1021, in finish\n    fin()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/tests/integration/ha_tests/conftest.py\", line 57, in loop_wait\n    await change_patroni_setting(ops_test, \"loop_wait\", initial_loop_wait)\nTypeError: change_patroni_setting() missing 1 required positional argument: 'password'","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1728610461939,"stop":1728610461939,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1728610461939,"stop":1728610461939,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1728610461940,"stop":1728610462294,"duration":354},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1728611548382,"stop":1728611548382,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"restart_policy","time":{"start":1728611548382,"stop":1728611648450,"duration":100068},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1728611548382,"stop":1728611548382,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"loop_wait","time":{"start":1728611648450,"stop":1728611650584,"duration":2134},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"This tests checks that a cluster recovers from a full cluster restart.\n\n    The test can be called a full cluster crash when the signal sent to the OS process\n    is SIGKILL.\n    ","status":"broken","statusMessage":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 97, in pytest_runtest_teardown\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 100, in pytest_runtest_teardown\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 853, in pytest_runtest_teardown\n    yield from self._runtest_for(item, \"teardown\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 885, in pytest_runtest_teardown\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 189, in pytest_runtest_teardown\n    item.session._setupstate.teardown_exact(nextitem)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 557, in teardown_exact\n    raise exceptions[0]\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 546, in teardown_exact\n    fin()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1032, in finish\n    raise exceptions[0]\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1021, in finish\n    fin()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/tests/integration/ha_tests/conftest.py\", line 57, in loop_wait\n    await change_patroni_setting(ops_test, \"loop_wait\", initial_loop_wait)\nTypeError: change_patroni_setting() missing 1 required positional argument: 'password'","steps":[],"attachments":[{"uid":"f2773f25ed573e0d","name":"log","source":"f2773f25ed573e0d.txt","type":"text/plain","size":2537}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"loop_wait::finalizer","time":{"start":1728611692602,"stop":1728611692602,"duration":0},"status":"broken","statusMessage":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'\n","statusTrace":"  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-k8s-operator/test-runners-2-azure-arm64-postgresql-k8s-operator/tests/integration/ha_tests/conftest.py\", line 57, in loop_wait\n    await change_patroni_setting(ops_test, \"loop_wait\", initial_loop_wait)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"restart_policy::finalizer","time":{"start":1728611692602,"stop":1728611692602,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes::finalizer","time":{"start":1728611692603,"stop":1728611693627,"duration":1024},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1728611693629,"stop":1728611693629,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1728612000154,"stop":1728612000798,"duration":644},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1728612000799,"stop":1728612000799,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1728612000799,"stop":1728612000799,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1728612000799,"stop":1728612000800,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"unstable"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"runner"},{"name":"thread","value":"14938-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"process","value":"'postgres'"},{"name":"signal","value":"'SIGKILL'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"7180d96c054f09ac","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":16,"skipped":0,"passed":14,"unknown":1,"total":31},"items":[{"uid":"331c2b885dd8c23f","reportUrl":"../106//#testresult/331c2b885dd8c23f","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1728525077828,"stop":1728525116880,"duration":39052}},{"uid":"77d59d6bb2e51e9c","reportUrl":"../105//#testresult/77d59d6bb2e51e9c","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1728438682834,"stop":1728438720305,"duration":37471}},{"uid":"a45e4ed6ca5caba5","reportUrl":"../104//#testresult/a45e4ed6ca5caba5","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1728352265154,"stop":1728352305694,"duration":40540}},{"uid":"d9b44d34fd0e94c8","reportUrl":"../103//#testresult/d9b44d34fd0e94c8","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1728265802116,"stop":1728265839075,"duration":36959}},{"uid":"db16783eb76c1de","reportUrl":"../102//#testresult/db16783eb76c1de","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1728179458679,"stop":1728179496208,"duration":37529}},{"uid":"e19ebb1692652ba1","reportUrl":"../101//#testresult/e19ebb1692652ba1","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1728093032420,"stop":1728093064553,"duration":32133}},{"uid":"cb73f4ef3cd4cf61","reportUrl":"../100//#testresult/cb73f4ef3cd4cf61","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1728006747915,"stop":1728006785849,"duration":37934}},{"uid":"af14d793d9510054","reportUrl":"../99//#testresult/af14d793d9510054","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1727920265602,"stop":1727920303384,"duration":37782}},{"uid":"314c230c6a3984b1","reportUrl":"../98//#testresult/314c230c6a3984b1","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1727833781684,"stop":1727833820024,"duration":38340}},{"uid":"406ffed5e56c73fd","reportUrl":"../97//#testresult/406ffed5e56c73fd","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1727747475625,"stop":1727747513018,"duration":37393}},{"uid":"96048d90086c0e2c","reportUrl":"../96//#testresult/96048d90086c0e2c","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1727661051084,"stop":1727661087421,"duration":36337}},{"uid":"258fa1084368c283","reportUrl":"../95//#testresult/258fa1084368c283","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1727574687514,"stop":1727574724229,"duration":36715}},{"uid":"f66667adec27a9ba","reportUrl":"../94//#testresult/f66667adec27a9ba","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1727488289097,"stop":1727488326742,"duration":37645}},{"uid":"250f0c1cf90e9f5a","reportUrl":"../93//#testresult/250f0c1cf90e9f5a","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1727401902798,"stop":1727401940340,"duration":37542}},{"uid":"21d237c6682945c0","reportUrl":"../92//#testresult/21d237c6682945c0","status":"broken","statusDetails":"TypeError: change_patroni_setting() missing 1 required positional argument: 'password'","time":{"start":1727315537971,"stop":1727315583283,"duration":45312}},{"uid":"e96450b78a17b832","reportUrl":"../90//#testresult/e96450b78a17b832","status":"passed","time":{"start":1722218526929,"stop":1722218556618,"duration":29689}},{"uid":"3bfcdd096f961df","reportUrl":"../89//#testresult/3bfcdd096f961df","status":"passed","time":{"start":1722132057551,"stop":1722132096293,"duration":38742}},{"uid":"1401fa5dfa11dfae","reportUrl":"../88//#testresult/1401fa5dfa11dfae","status":"passed","time":{"start":1722045987950,"stop":1722046017813,"duration":29863}},{"uid":"29a6923c0894692d","reportUrl":"../87//#testresult/29a6923c0894692d","status":"passed","time":{"start":1721959278792,"stop":1721959308572,"duration":29780}},{"uid":"2d8a4cf0213284ff","reportUrl":"../86//#testresult/2d8a4cf0213284ff","status":"passed","time":{"start":1721873005984,"stop":1721873041725,"duration":35741}}]},"tags":["unstable","asyncio","group(1)"]},"source":"ba850852bfa5b8ef.json","parameterValues":["'postgres'","'SIGKILL'"]}